mauth-client.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'rack'
   spec.add_dependency 'dice_bag', '>= 0.9', '< 2.0'
 
-  spec.add_development_dependency 'bundler', '~> 1.10'
+  spec.add_development_dependency 'bundler', '~> 2.0.1'
```

Comment:
Thanks for the PR!

Can you make that `'>= 1.17'` ?
- 2.0.1 is far too restrictive and going to be a maintenance burden
- not everyone is on 2.x yet
- from now on every christmas a new major bundler version is going to be released, hence `>=` and not for instance `~> 2.0`